tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -63,6 +63,15 @@ table-concurrency = 6
 # medium, this value might need to be adjusted for optimal performance.
 io-concurrency = 5
 
+# The maximum number of non-fatal errors to tolerate before stopping TiDB Lightning.
+# Non-fatal errors are localized to a few rows, and ignoring those rows allows the import process to continue.
+# Setting this to N means that TiDB Lightning will stop as soon as possible when the (N+1)-th error is encountered.
+# The skipped rows will be inserted into tables inside the "task info" schema on the target TiDB, which can be configured below.
+max-error = 0
+# task-info-schema-name is the name of the schema or database that stores TiDB Lightning execution results.
+# To disable error recording, set this to an empty string.
+# task-info-schema-name = 'lightning_task_info'
```

Comment:
Follow-up: To translate this part to Chinese

Reply:
See PR: https://github.com/pingcap/docs-cn/pull/8211